units/stats_host_idle.go

```diff
@@ -180,15 +180,20 @@ func (j *collectHostIdleDataJob) incrementCostForDuration(ctx context.Context) (
 }
 
 func (j *collectHostIdleDataJob) getHostStatsMessage(cost float64, idleTime time.Duration) message.Composer {
+	taskHost := j.host.User == evergreen.User && !j.host.SpawnOptions.SpawnedByTask
```

Comment:
The second clause is redundant, since they have the user set to the task that spawned them.